internal/util/cephcmds.go

```diff
@@ -32,6 +32,41 @@ import (
 // InvalidPoolID used to denote an invalid pool.
 const InvalidPoolID int64 = -1
 
+// ExecuteCommandWithNSEnter executes passed in program with args with nsenter
+// and returns separate stdout and stderr streams. In case ctx is not set to
+// context.TODO(), the command will be logged after it was executed.
+func ExecuteCommandWithNSEnter(ctx context.Context, netPath, program string, args ...string) (string, string, error) {
+	var (
+		sanitizedArgs = StripSecretInArgs(args)
+		stdoutBuf     bytes.Buffer
+		stderrBuf     bytes.Buffer
+	)
+	args = append([]string{fmt.Sprintf("--net=%s", netPath), program}, args...)
+	cmd := exec.Command("nsenter", args...) // #nosec:G204, commands executing not vulnerable.
```

Comment:
I would make it `nsenter --net=%s -- <program> <args>` to prevent `nsenter` from trying to interpret some program argument as an option.  It did that in the past and IIRC that got fixed but let's stay on the safe side.

Reply:
Thanks @idryomov sounds like a good idea, will update the PR